### ISSUE TITLE
The LibXML installation page is not up to date

### ISSUE THREAD

**Issue by mpepito13** [php/doc-en#1362]
From manual page: https://php.net/libxml.installation

The following configure option doesn't exist anymore : --with-libxml-dir
The following seems fine : --with-libxml

Thanks
---

**Comment by cmb69** [comment]
Right. As of PHP 7.4.0, libxml relies on pkg-config to find the libxml installation; previously, the installation was as described in the PHP manual.
